### DISCUSSION TITLE
Dúvida

### DISCUSSION THREAD

**Discussion by PedroCacador** [alyssapiresfernandescefet/openrpg#14] [Q&A]
Como posso alterar a fonte?

**Comment by lucaspiresfernandes** [comment]
Olá! Acredito que você esteja falando das fontes personalizadas do retrato, então vamos lá:

Para aplicar suas fontes personalizadas no retrato, você deve primeiro baixar uma fonte (pode ser qualquer formato de fonte: .ttf, .otf, etc.)

Quando já baixado, você acessa as configurações na conta do mestre e vai na aba "Retrato". No final, tem uma categoria de fontes personalizadas. Você clica ali e seleciona a fonte que quer enviar.

Depois de enviado, verá que o nome da fonte ficará gravada. Recarregue seus retratos e prontinho!

**Comment by ptmgkira** [comment]
Ola Boa noite estou tentando acessar a pagina main,depos de ter feito todo o precesso pela heroku db e deploy e quando vou acessar o link aparece o codigo 500 e acaba não saindo disso e nem acessando outra paginas dentro dele.

**Comment by lucaspiresfernandes** [comment]
@ptmgkira me chama no Discord: That Alyssa#0615
